docs/integration_resources/software/nuke.md

```diff
@@ -14,61 +14,134 @@ nshift.show()
+| *Instance*   | -              | Knob                                             | EvalString_Knob                                              | Requires custom implementation                                                                                                                                                                                                                            |
+| *Int*        | -              | Knob                                             | Int_Knob                                                     |                                                                                                                                                                                                                                                           |
+| *List*       | points*        | Input<br/>Knob                                   | - <br/>Bool_knob                                             | Requires to connect a Tracker node to get the points from. A Knob to active auto inversion of Y coordinate is added. This inversion is useful to make the point coordinates match the image instances coordinates with the (0, 0) at the top left corner. |
+| *List*       | -              | Knob                                             | EvalString_Knob                                              | Requires custom implementation                                                                                                                                                                                                                            |
+| *Object*     | -              | Knob                                             | EvalString_Knob                                              | Requires custom implementation                                                                                                                                                                                                                            |
+| *String*     | -              | Knob                                             | EvalString_Knob                                              |                                                                                                                                                                                                                                                           |
+
+>[!NOTE]
+> * The Plugs to Knobs relations with a SPlug Code required means that the name of the plug need to start by that name, but can be longer. For example you can have more than 1 image with Shift Plug: "image1", "image2",...

```

Comment:
Asterisk displays as dotted list in the website, please fix that.

Here is a suggestion for the text, please commit this suggestion prior to the tests for the asterisk display.
```suggestion
>[!NOTE]
> * The conversion for certain Shift Plugs to Nuke Knobs will change depending on the Plug naming protocol. Specifically the shiftWorkflow node plugin for Nuke will identify as "special", plugs that start with the word defined in the `SPlug Code` section of the table above, e.g. "image1", "image2",... or "imageSequenceOriginal", "imageSequenceMasked",...

```